### ISSUE TITLE
"deprecated" function in the rule copying code

### ISSUE THREAD

**Issue by nikmat1** [swaschkut/pan-os-php#18]
When rules are copied from one device group to another and the destination device group already has a rule with the name identical to one being copied a message as follows appears:

```
...
   - object '<name of the rule>'  passing through Action='copy' Args: location=<destination device group>, preORpost=post,
PHP Deprecated:  Increment on non-alphanumeric string is deprecated in /tools/pan-os-php/lib/rule-classes/RuleStore.php on line 534 
...
```

The script continues copying after the message (which is good!), so I don't know how serious this problem might be.

**Comment by swaschkut** [comment]
thanks for this finding.

there are now two different develop containers available:

**docker push swaschkut/pan-os-php:develop**

and as discussed one with ubuntu 24.0.10
**docker push swaschkut/pan-os-php:develop_ubuntu24_10**

for your information,
my email was blocked during send,
check your slack, there is a PDF available for you

**Comment by swaschkut** [comment]
fixed in version 2.1.25
#21
